pkg/minikube/download/preload.go

```diff
@@ -45,6 +45,15 @@ const (
 	PreloadVersion = "v11"
 	// PreloadBucket is the name of the GCS bucket where preloaded volume tarballs exist
 	PreloadBucket = "minikube-preloaded-volume-tarballs"
+
+	// Enumeration for preload existence cache.
+	preloadExistsUNKNOWN = 0 // Value when preload status has not been checked.
```

Comment:
`iota` seems like the better option here instead of 0, 1, and 2 directly. Makes it easier to maintain later on.
also I think we can agree these names are less than ideal. what about preload* instead of preloadExists* for them? it will still be pretty clear what's going and will be less awkward to read.

Reply:
Ahhh ok I remember reading something about iota but I don't think those examples were good. I now understand and yes that is way better. I agree with preload* and have done accordingly. @medyagh also recommended removing the all caps, since that's not really the style in our codebase. Finally I had to rename preloadExists to preloadPresent because having preloadExists and PreloadExists does not sound fun :P

Done.